src/common/state/ExpressionList.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-  <Attribute name="ExpressionList" purpose="This class contains a list of expressions and some functions to manipulate them." persistent="true" exportAPI="STATE_API" exportInclude="state_exports.h" codefile="ExpressionList.code">
+  <Attribute name="ExpressionList" purpose="This class contains a list of expressions and some functions to manipulate them." persistent="true" keyframe="true" exportAPI="STATE_API" exportInclude="state_exports.h" codefile="ExpressionList.code">
```

Comment:
I'm concerned about the addition of 'keyframe=true', is this needed?

Reply:
I don't think so. I noticed that popped up when I re-generated. I'll go ahead and remove that.